.circleci/config.yml

```diff
@@ -35,7 +39,11 @@ jobs:
 
       - save_cache:
           paths:
-            - ~/.gradle
+            - ~/.gradle/wrapper
+          key: gradle-wrapper-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
```

Comment:
I've split the cache into two which seems to be faster due to the smaller resulting archive sizes. This will also increase how much we get from a cache in each job as we'll be changing dependencies much more than we change Gradle versions.